.github/dependabot.yml

```diff
@@ -14,6 +14,19 @@ updates:
       - "raft review"
     reviewers:
       - "raft-tech/tdrs-developers"
+    ignore:
+      - dependency-name: "redux*"
+        update-types: ["version-update:semver-patch"]
```

Comment:
Might be useful to add a link to [this article](https://github.blog/changelog/2021-05-21-dependabot-version-updates-can-now-ignore-major-minor-patch-releases/) if this file accepts comments.

Reply:
Ok, will add a standard yaml comment. Hopefully, dependabot takes it.